methods.py

```diff
@@ -598,7 +598,7 @@ def detect_darwin_sdk_path(platform, env):
             sdk_path = decode_utf8(subprocess.check_output(['xcrun', '--sdk', sdk_name, '--show-sdk-path']).strip())
             if sdk_path:
                 env[var_name] = sdk_path
-        except (subprocess.CalledProcessError, OSError) as e:
+        except (subprocess.CalledProcessError, OSError) as _:
```

Comment:
Can't we just drop the ` as e` part if we don't need it?

Reply:
I think its safe to delete, I personally would like to do `print(e)` just to see what the error is.
Although I have removed this like you suggested in the latest PR.